docs/en_US/CommunitySharings/AutoCompletion.rst

```diff
@@ -1,57 +0,0 @@
-Auto Completion for nnictl Commands
-===================================
-
```

Comment:
Do we have a future plan to support this feature? If yes, we can add a ``.. warning`` here that states the feature is broken in current releases and will be fixed in feature.

Reply:
It seems that @liuzhe-lz is the owner of this feature. - would you mind tell us the future plan?

Reply:
It is planned, with very priority.

Reply:
This feature has been deferred since v2.0, and I don’t know when it will be implemented, so I temporarily removed it...